src/nwscript/decompiler.cpp

```diff
@@ -38,7 +38,7 @@ void Decompiler::createNSS(Common::WriteStream &out) {
 	_ncs->analyzeStack();
 	_ncs->analyzeControlFlow();
 
-	out.writeString("// Decompiled using ncsdecomp");
+	out.writeString("// Decompiled using ncsdecomp\n");
```

Comment:
That should probably be its own commit

Reply:
Actually, i accidentally inserted this into the commit, i will remove this.